File: BD/laba4/DBAdmin/DBAdmin/entity/ApartmentBuilding.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace DBAdmin.entity
{
    [Table("apartment_buildings")]
    public class ApartmentBuilding
    {
        [Column("id")]
        public int Id { get; set; }

        [Column("city")]
        public string City { get; set; }

        [Column("street")]
        public string Street { get; set; }

        [Column("house_number")]
        public int HouseNumber { get; set; }
    }
}

File: BD/TestDb/TestDb/ApartmentBuilding.py
from sqlalchemy import Column, Integer, String
from sqlalchemy.orm import relationship
from MyBaseModel import *

class ApartmentBuilding(MyBaseModel):
    __tablename__ = "apartment_buildings"

    id = Column(Integer, primary_key=True, index=True)
    city = Column(String)
    street = Column(String)
    house_number = Column(Integer)
    requests_client_applications = relationship("ClientApplication", back_populates="apartment_buildings")

    def __str__(self):
        return f"{self.city}, {self.street}, д.{self.house_number}"